docs/nginx-ingress.md

```diff
@@ -131,3 +131,58 @@ spec:
                 port:
                   number: 9443
 ```
+
+### Replication configuration:
```

Comment:
This is not necessarly related to replication, this is mainly a very slow deployment or best practice were not followed, like a user storing a lot of objects under a single prefix

the one here works most of the time, 

https://min.io/docs/minio/linux/integrations/setup-nginx-proxy-with-minio.html